tools/js-sdk-release-tools/src/common/changelog/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -3,24 +3,34 @@ import { updateUserAgent } from "../../xlc/codeUpdate/updateUserAgent.js";
 
 import fs from 'fs';
 import * as path from 'path';
-
+import { getSDKType} from "../utils.js";
 const todayDate = new Date();
 const dd = String(todayDate.getDate()).padStart(2, '0');
 const mm = String(todayDate.getMonth() + 1).padStart(2, '0'); //January is 0!
 const yyyy = todayDate.getFullYear();
 
 const date = yyyy + '-' + mm + '-' + dd;
 
-export function makeChangesForFirstRelease(packageFolderPath: string, isStableRelease: boolean) {
+export function getFirstReleaseContent(packageFolderPath: string, isStableRelease: boolean) {
     const packageJsonData: any = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'package.json'), 'utf8'));
+    const sdkType = getSDKType(packageFolderPath);
+    const isModularClient = (sdkType === 'ModularClient')? true : false;
+    const firstBetaAndModularClientContent = `Initial release of the ${packageJsonData.name} package`;
+    const firstStableAndModularClientContent = `This is the first stable version with the package of ${packageJsonData.name}`;
+    const defaultContent = `The package of ${packageJsonData.name} is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart).`
```

Comment:
I found the the link contins `mgmt`, please double confirm it's correct. Though modular is mainly used for management plane, it can be used in data plane as far as I know.

Reply:
Please double confirm, looks like this is not exactly the same in the issue

Reply:
This is for modularClient, whether it is a data plane or an mgmt plane

Reply:
@MaryGao  could you double confirm this is what you want?

Reply:
/cc @kazrael2119